notification-alert.tf

```diff
@@ -0,0 +1,21 @@
+// single alert to minify unnecessary cost because threshold used in here is minimal
+module "reform-scan-notification-alert" {
+  source            = "git@github.com:hmcts/cnp-module-metric-alert"
+  location          = "${azurerm_application_insights.appinsights.location}"
+  app_insights_name = "${azurerm_application_insights.appinsights.name}"
+
+  enabled    = "${var.env == "prod"}"
+  alert_name = "Reform_Scan_notification"
+  alert_desc = "Triggers when notification service sends at least 5 notifications within a 30 minutes window timeframe."
+
+  app_insights_query = "traces | where message startswith 'Sending error notification'"
```

Comment:
worth adding comment in application code stating that this log is used in the alert (so that it doesn't get changed accidentally breaking this notification)

Reply:
PR: https://github.com/hmcts/reform-scan-notification-service/pull/198

Reply:
There's a 2h delay between receiving notification to send and actually sending it.
We want to be notified ASAP.

Reply:
should it not be based on error notification saved in db (or parsed from servicebus) instead of sending? after a delay and fact that we want to be notified before the actual sending is done this alert is not serving the purpose

Reply:
Now based on receiving notification rather than sending it

Reply:
There is also an option to base this notification on successfully handling the notification ('Handle notification message successful' in NotificationMessageHandler) but alerting upon receiving the message is quicker